File: armstrong_number/armstrong.py
num = int(input())

arm = 0
temp = num
while (num > 0):
    mod = num % 10
    arm += mod ** 3     # pow(mod, 3)
    
    num = num // 10

if (temp == arm):
    print(f"{temp} is an notArmstrong number")
else:
    print(f"{temp} is not an Armstrong number")

File: armstrong_number/armstrong.c
#include <stdio.h>

int main()
{
    int num, mod, arm, temp;
    arm = 0;

    scanf("%d", &num);
    
    temp = num;
    while (num > 0)
    {
        mod = num % 10;
        arm += mod * mod * mod;

        num = num / 10;
    }

    if (temp == arm)
    {
        printf("%d is an Armstrong number\n", temp);
    }
    else
    {
        printf("%d is not an Armstrong number\n", temp);
    }
    return 0;
}